frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -316,6 +316,10 @@ func toDispatchState(ctx context.Context, dt []byte, opt ConvertOpt) (*dispatchS
 		}
 
 		if v := st.Platform; v != "" {
+			// Mark whether the specified platform is redudant for the linter
```

Comment:
"redundant"

Reply:
Removing this line due to another section of the code review.